doc/languages-frameworks/texlive.section.md

```diff
@@ -28,6 +28,29 @@ Since release 15.09 there is a new TeX Live packaging that lives entirely under
   }
   ```
 
+- To configure the default page size, pass `paperSize`:
+
+  ```nix
+  texlive.combine {
+    # inherit (texlive) whatever-you-want;
+    paperSize = "letter"; # a4 or letter
+  }
+  ```
+
+  The argument `paperSize` can also be an attribute set, specifying different page sizes for different programs:
+
+  ```nix
+  texlive.combine {
+    # inherit (texlive) whatever-you-want;
+    paperSize = {
+      "dvipdfmx" = "legal";
+      "psutils" = "letter";
+    };
```

Comment:
How do I set the default without trying to list every program in this case?

Reply:
Is this even practical?

Reply:
Special-casing the `default`-attribute to also run `paper_all` is definitely possible. I can't really imagine a usecase for this, but it's probably more common than wanting to specify pairwise distinct paper sizes for all the programs (there can't be that many sizes, right?)

Reply:
Right. I was exposing the tlmgr API to the letter, but tlmgr can be run twice and `combine` cannot. I think `default` is very unambiguous and I can't think of anything better (`all` is ambiguous, `_default` is a bit ugly).

I agree that this is a fairly abstract concern – even Debian only offers a4/letter I think. Hence the `paperSize = "letter"` version.

(xdvi has tons of sizes for some reason! most of the others are just a4/letter.)

Reply:
Conclusion for today: `paper_all` returns an error when some of the packages are not installed, even though everything goes well. We should not ignore errors as they could represent actual failures, so we have to list the relevant packages by hand and configure only the ones that are installed.

I say, let's focus on `texlive.buildEnv` and come back to this later.